### DISCUSSION TITLE
Missing MvvmCross.Platforms.Ios namespace

### DISCUSSION THREAD

**Discussion by Leo506** [MvvmCross/MvvmCross#4831] [Q&A] [answered]
After upgrading `MvvmCross` from 9.1.1 to 9.2.0 my iOS project lost `MvvmCross.Platforms.Ios` namespace.
Project runs on .Net 8 (target framework `net8.0-ios`, `SupportedOSPlatformVersion `- `15.0`)

**Comment by Cheesebaron** [comment, marked as answer]
Make sure to update your .NET workloads:

```
sudo dotnet workloads update
```

**Reply by Leo506** [reply]
It's works, thanks. I needed to restore workloads and update XCode
